README.md

```diff
@@ -1 +1,13 @@
-# micro-floating
\ No newline at end of file
+# Project: Position and floating with CSS3
+
+## Description
```

Comment:
You can still make the readme file look more professional, find inspiration [here](https://github.com/othneildrew/Best-README-Template) or [here](https://www.udacity.com/course/writing-readmes--ud777).